react-events/my-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  let handleCustomClick = (text) => {
+    window.alert(text);
+  };
+
+  return (
+    <div className="App">
+      <CustomButton
+        text={'First Button'}
+        style={{ color: 'red' }}
```

Comment:
Instead of passing `style` as the prop, I would just do `color`. `style` is kinda confusing because you use it to apply styles to an element, but you can't do that to a component. So it's better to use a different prop name.

You also technically aren't even accessing this prop inside `CustomButton`, because you reference `prop.color`, but only `prop.style` exists